src/NetTopologySuite/Geometries/Implementation/PackedCoordinateSequence.cs

```diff
@@ -375,7 +375,7 @@ public PackedFloatCoordinateSequence(float[] coords, int dimension, int measures
             : base(coords?.Length / dimension ?? 0, dimension, measures)
         {
             if (coords == null)
-                coords = new float[0];
+                coords = Array.Empty<float>();
```

Comment:
:page_facing_up: Same basic note about behavior change as in `MinimumBoundingCircle.cs`